src/components/Select/Select.tsx

```diff
@@ -85,18 +95,22 @@ export default class Select extends React.Component<
 
         return (
             <select
+                name={name}
                 {...selectProps}
-                onBlur={(e) => this.onSelectChange(e, onBlur)}
-                onChange={(e) => this.onSelectChange(e, onChange)}
+                {...attributes}
+                ref={ref}
+                onBlur={(e) => onSelectChange(e, onBlur)}
+                onChange={(e) => onSelectChange(e, onChange)}
             >
                 {React.Children.map(children, (child, key) =>
                     React.cloneElement(child as React.ReactElement<any>, {
                         "aria-selected":
-                            children[key].props.children ===
-                            this.state.selectedOption,
+                            children[key].props.children === selectedOption,
```

Comment:
Perhaps a better question for @scooooooooby : 
So from the stories, we're supposed to use this like 
```
<Select>
      <option aria-selected={true}>
                {text("Option One", optionsGroup[0])}
            </option>
</Select>
```

but is ` aria-selected={true}` actually doing anything?  Isn't this line overriding `aria-selected={true}` or is something else going on here?

Reply:
I was a bit unsure about this at first, but then I noticed that it was updating `aria-selected` correctly. I can't say I fully know how this works.

Reply:
`aria-selected` that you pass in the `option` sets the first selected state that gets updated onChange from the user, if I'm remembering correctly.

Our a11y checker gets upset if they're not applied to all of the `option`s, which when we moved to passing them in as children I had to add manually... I'm sure someone smarter than me could figure out how to automate it for the consumer with `React.cloneElement` though 😂